### PR TITLE
Add missing interpolator

### PR DESCRIPTION
**Pull request by alexandrustana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ import com.lorandszakacs.enclosure.Enclosure
 
 object Printer {
   def locatedPrintln(s: String)(implicit enc: Enclosure): Unit =  {
-    println(s"[${enc.fullModuleName}] s")
+    println(s"[${enc.fullModuleName}] $s")
```
Comment (lorandszakacs) [inline README.md:33]: this breaks scaladoc. need to find out how to insert literal '$' char :grimacing:

Comment (lorandszakacs) [reply README.md:33]: `Compile / doc` in sbt to see why.

Comment (alexandrustana) [reply README.md:33]: `$$` seems to fix this, I've changed it in `Enclosure.scala` and sbt seems to embrace this 😅

Comment (lorandszakacs) [reply README.md:33]: great. Thank you! :+1:

**Pull request merged**